Sources/YapDB.swift

```diff
@@ -97,15 +103,15 @@ extension YapDB {
         - parameter database: a YapDatabase instance
         - returns: A Bool
         */
-        public func isRegisteredInDatabase(database: YapDatabase) -> Bool {
-            return (database.registeredExtension(name) as? YapDatabaseView) != .None
+        open func isRegisteredInDatabase(_ database: YapDatabase) -> Bool {
```

Comment:
Changed this to `public`. And I think there's actually a bug on line 107, I think it should be `YapDatabaseExtension` not `YapDatabaseView`. 76a94a9